problems/isOdd.js

```diff
@@ -6,9 +6,15 @@
 * @returns {boolean} - True if n is odd, false if n is even, or not a number.
 
 */
+//Can't get the even to pass
+function isOdd(num) {
+   if(num % 2 === 0) {
```

Comment:
Very close on this one! If you return num % 2 === 1 you would get it right. That would return true for odds, and false for everything else.